Задания с 28.09.2021/28.09.2021-Task1/28.09.2021-Task1/BinaryRepresentation.c

```diff
@@ -0,0 +1,123 @@
+﻿#include <stdio.h>
+#include <stdlib.h>
+#include <locale.h>
+#include <stdbool.h>
+#include <math.h>
+
+#define INT_BITS_COUNT sizeof(int) * 8
```

Comment:
```suggestion
#define INT_BITS_COUNT (sizeof(int) * 8)
```
В макросах лучше вообще всегда скобки ставить, мы же не знаем, в каком контексте выполняется подстановка. Простой пример:
```c
#define TEST 2 + 2
...
TEST * 2
```
Кажется, что будет 8, а на самом деле будет 6